.pre-commit-config.yaml

```diff
@@ -11,4 +11,10 @@ repos:
 -   repo: https://github.com/pre-commit/mirrors-mypy.git
     rev: v0.790
     hooks:
-    -   id: mypy
+    - id: mypy
+-   repo: https://github.com/pycqa/isort
+    rev: 5.7.0
+    hooks:
+    - id: isort
+      args: ["--profile", "black"]
```

Comment:
to resolve conflict between black and isort 

https://github.com/PyCQA/isort/issues/1518#issuecomment-703450843